lib/ui/common/Dogtag.dart

```diff
@@ -21,37 +21,40 @@ class DogTagStyle {
   final int maxLines;
   final double iconSize;
   final double spacing;
+  final Color iconColor;
 
-  DogTagStyle(
+  const DogTagStyle(
       {this.backgroundColor,
       this.borderRadius,
       this.edgePadding,
       this.titleTextStyle,
       this.maxLines,
       this.iconSize,
+      this.iconColor,
       this.spacing});
 
   factory DogTagStyle.defaultStyle() {
     return DogTagStyle(
         backgroundColor: Color(0x1425df0c),
         borderRadius: BorderRadius.all(Radius.circular(20.0)),
-        edgePadding: EdgeInsets.only(top: 8.5, right: 12, left: 12, bottom: 8),
+        edgePadding:
+            EdgeInsets.only(top: 4.5, right: 12, left: 12, bottom: 4.5),
         titleTextStyle: TextStyle(
             fontSize: 15,
-            fontWeight: FontWeight.normal,
-            color: Color(0xff25df0c)),
+            fontWeight: FontWeight.w500,
+            color: Color(0xff21c400)),
         maxLines: 1,
         iconSize: 8,
+        iconColor: Colors.black,
         spacing: 5.5);
   }
 
   factory DogTagStyle.negativeStyle() {
```

Comment:
We should only have 1 private default style in the widget. All. Other styles should be outside the widget at the point of use.